frontend/src/scenes/plugins/tabs/apps/AppView.tsx

```diff
@@ -151,6 +161,69 @@ export function AppView({
                             </LemonButton>
                         )}
 
+                        {canGloballyManagePlugins(user?.organization) && (
+                            <>
+                                {unusedPlugins.includes(plugin.id) && (
```

Comment:
nitpick feel free to ignore: someone might waste time trying to find the uninstall button if it's hidden. It might be better to show it in a disabled state, with a popup of "cannot uninstall app still in use"?